explorer/src/Pos/Explorer/Web/Server.hs

```diff
@@ -13,6 +16,15 @@ module Pos.Explorer.Web.Server
        , explorerApp
        , explorerHandlers
 
+       -- pure functions
+       , pureGetBlocksTotal
+       , pureGetBlocksPagesTotal
+
```

Comment:
I know that naming might be hard. But I'm sure there are better ways to name pure function without `pure...`.

Reply:
Why? It's the most simple and straightforward thing to write, so why not use that? Note that the original function names are taken, and these are used in tests, so I don't see why it shouldn't be named like this.
But if you have better ideas how to express that a function is pure without using the word pure...
The alternative would be to use something like `getBlocksTotal` for a pure function and `getBlocksTotalM` for a "monadic" one. But then we would need to change the other functions as well.

Reply:
> Why?
Because we have tons of pure functions which are not prefixed with `pure`. So why should we do this with few of them? You could use a name which described what the function does. For example `roundBlocksPagesTotal` instead of `pureGetBlocksPagesTotal`.